libbeat/dashboards/kibana_loader.go

```diff
@@ -153,6 +157,6 @@ func (loader KibanaLoader) statusMsg(msg string, a ...interface{}) {
 	if loader.msgOutputter != nil {
 		loader.msgOutputter(msg, a...)
 	} else {
-		logp.Debug("dashboards", msg, a...)
+		loader.defaultLogger.Debugf("%s %v", msg, a)
```

Comment:
the old implementation looks like "statusMsg" implements a Printf like interface (better named would have been `statusMsgf`. 

`s/loader.defaultLogger.Debugf("%s %v", msg, a)/loader.defaultLogger.debugf(msg, a...)/`

Is MsgOutputer used somewhere? It kind of looks like a 'duplicate' interface for logging.

Reply:
Yah, it was one of those things where whatever calls the loader also asks for that message handler option, so it seems to go fairly high up the call stack and I was hesitant to mess with it.

Reply:
I see.

Let's change this line to this (to restore the behavior from before this PR):

```
loader.defaultLogger.Debugf(msg, a...)
```